Intro_markdown.Rmd

```diff
@@ -156,7 +156,7 @@ getwd()
 If you want to change the working directory to a specific repository/folder, you can use the `setwd()` command as follows:
 
 ```{r}
-setwd("~/folder_name")
```

Comment:
not sure about this one - was just making it consistent with the participant code

Reply:
See above comment

Reply:
Ah I think you might have been referring to [this](https://github.com/moj-analytical-services/IntroRTraining/pull/49#discussion_r1234985708)

Reply:
@ian-rickard sorry I think the one below to the participant code

Reply:
ok, I hope I've taken care of it all